plugin/parser/engine/pg/test-data/test_type_data.yaml

```diff
@@ -0,0 +1,4 @@
+- stmt: drop type if exists tp1, public.tp2;
```

Comment:
Add comprehensive tests about a table using an enum type?

Reply:
This makes no difference to the parser about enum types and built-in types. The create table statement only know the type as the type name.

Reply:
I added the test in the differ test